customizations/mei-basic.xml

```diff
@@ -11,6 +12,10 @@
                     <resp>Authored by</resp>
                     <name xml:id="JK">Johannes Kepper</name>
                 </respStmt>
+                <respStmt>
+                    <resp>Revisions</resp>
+                    <name xml:id="LP">Laurent Pugin</name>
+                </respStmt>
```

Comment:
Didn't you just revise it as well? Good catch, but we should be more generous / correct when giving credit…

Reply:
No, I just fixed the bugs. :-D

Reply:
The credit is also to "The MEI Community" so I'm covered by that.